frontend/test/__support__/mocks.js

```diff
@@ -34,21 +34,8 @@ jest.mock("ace/snippets/json", () => {}, { virtual: true });
 jest.mock("ace/snippets/json", () => {}, { virtual: true });
 jest.mock("ace/ext-language_tools", () => {}, { virtual: true });
 
-// Use test versions of components that are normally rendered to document root or use unsupported browser APIs
-import * as modal from "metabase/components/Modal";
-modal.default = modal.TestModal;
-
-import * as tooltip from "metabase/components/Tooltip";
-tooltip.default = tooltip.TestTooltip;
-
 jest.mock("metabase/components/Popover");
 
-import * as bodyComponent from "metabase/components/BodyComponent";
-bodyComponent.default = bodyComponent.TestBodyComponent;
-
-import * as table from "metabase/visualizations/visualizations/Table";
-table.default = table.TestTable;
-
```

Comment:
I wasn't sure if any of these mocks were actually used anywhere, but:
1. I couldn't find any reference
2. All tests are passing without them

If I've overlooked something, please let me know and I can easily revert this commit. It's not crucial for Enzyme removal.